app/models/xccdf/benchmark_searching.rb

```diff
@@ -48,6 +54,22 @@ def os_major_version_search(_filter, operator, value)
           parameter: [os_major_version_like_condition(value)] }
       end
 
+      def supported_os_minor_clauses(minor_versions)
+        minor_versions = [minor_versions].flatten.map(&:to_s)
+
+        SupportedSsg.latest_map.map do |major, major_ssgs|
+          ssg_versions = minor_versions.map do |minor_version|
+            major_ssgs[minor_version]&.version
+          end.compact
```

Comment:
Nit: we're sending non-unique minor versions in some cases. It doesn't affect the query result, but we might want to add a `.uniq` here.

```rb
[8] pry(main)> Xccdf::Benchmark.os_major_version(7).supported_os_minor_versions([7, 8, 9]).count
   (0.5ms)  SELECT COUNT(*) FROM "benchmarks" WHERE "benchmarks"."ref_id" ILIKE '%RHEL-7' AND ((benchmarks.ref_id LIKE '%RHEL-6' AND benchmarks.version IN ('0.1.21','0.1.28','0.1.28')) OR (benchmarks.ref_id LIKE '%RHEL-7' AND benchmarks.version IN ('0.1.43','0.1.46','0.1.52')))
```

The `benchmarks.version IN ('0.1.21','0.1.28','0.1.28')` part.

Reply:
:+1: